docs/public/theme.css

```diff
@@ -7,62 +7,68 @@
     'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', Monaco,
     'Courier New', Courier, monospace;
 
-  --color-white: #fff;
-  --color-black: #000014;
+  --color-base-white: 0, 0%;
```

Comment:
I would add a comment up here that explains what these numbers mean. 

For example: `--color-base` defines `hue, saturation` to be used as `hsla` colors.